src/main/scala-2.11/compat.scala

```diff
@@ -0,0 +1,8 @@
+package courier
+
+object Compat {
+  def asJava[T](set: Set[T]): java.util.Set[T] = {
+    import collection.JavaConverters._
+    set.asJava
+  }
+}
```

Comment:
You might need to tweak an editor setting to make sure the ending newline makes it in.